debian/control

```diff
@@ -144,7 +147,10 @@ Description: debugging symbols for ceph-mds
 
 Package: ceph-mgr
 Architecture: linux-any
-Depends: ceph-base (= ${binary:Version}), ${misc:Depends}, ${shlibs:Depends}
+Depends: ceph-base (= ${binary:Version}),
+         ${misc:Depends},
+         ${python:Depends},
```

Comment:
we don't run ceph-mgr through dh_python2 or dh_python3 in `debian/rules` yet. but since we install `src/pybind/mgr`, probably we should update `override_dh_python3` and `override_dh_python2` accordingly?